requirements/edx/base.txt

```diff
@@ -51,7 +51,9 @@ edx-lint==0.4.3
 astroid==1.3.8
 edx-django-oauth2-provider==1.1.4
 edx-django-sites-extensions==2.1.1
-edx-enterprise==0.19.1
+# enterprise app with desired changes
+-e git+https://github.com/edx/edx-enterprise.git@e0be788a74fbbce8be04a75b2ddca8843db17426#egg=edx-enterprise
```

Comment:
Given that the changes to edx-enterprise don't actually appear to depend on the changes here, other than the migration to `social-app-django` itself, can you go ahead and create that PR so that we can have time to review it appropriately, and be ready to merge and have a version bump when it's time to merge this?